lms/djangoapps/verify_student/tests/test_models.py

```diff
@@ -185,6 +185,7 @@ def create_and_submit(self):
         user = UserFactory.create()
         attempt = SoftwareSecurePhotoVerification(user=user)
         user.profile.name = u"Rust\u01B4"
+        user.username = u"Rust"
```

Comment:
Ii assume `user` object has already have a `username` which you can concatenate in the `logger.check`